coderd/database/types.go

```diff
@@ -35,6 +35,40 @@ type NotificationsSettings struct {
 	NotifierPaused bool      `db:"notifier_paused" json:"notifier_paused"`
 }
 
+type InboxNotificationReadStatus string
+
+const (
+	InboxNotificationReadStatusRead   InboxNotificationReadStatus = "READ"
```

Comment:
Rather create a SQL type for this to get automatic codegen
Nit: make this lowercase to match other types' values